frontend/src/app/shared/components/principal/principal-renderer.service.ts

```diff
@@ -6,12 +6,21 @@ import idFromLink from 'core-app/features/hal/helpers/id-from-link';
 import { IPrincipal } from 'core-app/core/state/principals/principal.model';
 import { PrincipalLike } from './principal-types';
 import { hrefFromPrincipal, PrincipalType, typeFromHref } from './principal-helper';
+import { PortalOutletTarget } from 'core-app/shared/components/modal/portal-outlet-target.enum';
 
 export type AvatarSize = 'default'|'medium'|'mini';
 
+export interface HoverCardOptions {
+  url?:string;
+  modalTarget?:PortalOutletTarget;
+  closeDelay?:number;
```

Comment:
I think I can drop the alignment. It seemed useful in an earlier iteration where I wanted to stay closer to the Figma design and center the hover card over the avatar. However, it doesn't look as good as I figured it should and later on I decided to treat WP and user hover cards similar.

The closeDelay has a similar reasoning. I wanted the user hover card to linger around longer since it's easy to accidentally leave the avatar on many pages. At the same time, I did not want to change WP hover card behavior: hence a new option.

I could just increase the closeDelay for both WP and user hover cards and remove this complexity. That would probably be best 🤔

Reply:
We discussed this and decided to treat WP and user hover cards the same. I have removed both options to reduce the complexity 👍🏻